src/components/ControlGroup/EnumControl.tsx

```diff
@@ -0,0 +1,24 @@
+type Props = {
+	options: Array<string>;
+	description?: string;
+	setPosition: (position: string) => void;
```

Comment:
LGTM please just rename `setPosition` to `onChange` for two reasons:

1. Event based callbacks should stick to web-standard naming conventions of `on<Event>`
2. Elements that mimic input based controls should use input based event names
3. `onClick` is also acceptable.